File: domain/src/main/java/com/sof/domain/Meeting.java
package com.sof.domain;

import lombok.*;

import javax.persistence.*;
import java.time.LocalDate;
import java.time.LocalTime;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class Meeting {

    private Long id;
    private Medecin medecin;
    private Patient patient;
    private Consultation consultation;

    private LocalDate date;
    private LocalTime heure;

}

File: front-client/src/main/angular/medical/src/app/models/meeting.ts

import {Medecin} from './medecin'
import {Patient} from './patient'
import {Consultation} from './consultation'

export interface Meeting {
  id?: string;
  medecin: Medecin;
  patient: Patient;
  consultation?: Consultation;
  date: string;
  heure: string;
}
